src/models/Call.ts

```diff
@@ -77,13 +80,7 @@ const waitForEvent = async (
 };
 
 export enum ConnectionState {
-    // Widget related states that are equivalent to disconnected,
-    // but hold additional information about the state of the widget.
-    Lobby = "lobby",
-    WidgetLoading = "widget_loading",
     Disconnected = "disconnected",
-
-    Connecting = "connecting",
     Connected = "connected",
     Disconnecting = "disconnecting",
```

Comment:
why do we drop the connecting state but not the disconnecting? 
I think it still could be useful to display the connecting state in the room list tile.
Is there any way we can compute this indirectly?
But I see, that we can still tell if its connected or not by using the presented. (which is the more important state)

Reply:
I would reintroduce the 'connecting' state in a future task, maybe when we're implementing the new DM call designs. It would need even more changes from EC presumably so I'm not inclined to squeeze it in here.